daffodil-core/src/main/scala/org/apache/daffodil/dsom/Facets.scala

```diff
@@ -22,7 +22,7 @@ import scala.xml.Node
 import org.apache.daffodil.exceptions.Assert
 import org.apache.daffodil.dpath.NodeInfo.PrimType
 
-trait Facets { self: RestrictionFactory =>
+trait Facets { self: Restriction =>
```

Comment:
The above is the only content change in this file. The rest is formatting of code per style guide.